frontend/pages/waysToSpendRoute/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Route } from '@shopgate/pwa-common/components';
+import { useTheme } from '@shopgate/engage/core';
+import SmilePanel from '../../components/SmilePanel';
+import SmileJoinFooter from '../../components/SmileJoinFooter';
+import getConfig from '../../helpers/getConfig';
+import { WAYS_TO_SPEND_ROUTE } from '../../constants';
+import connect from './connector';
+
+/**
+ * @returns {JSX}
+ */
+const waysToSpendRoute = ({ options }) => {
+  if (!options) {
+    return null;
```

Comment:
we should show some message here, otherwise the user will just see a blank empty page

Reply:
Added a message and loading indicator